mewarpx/mewarpx/mwxrun.py

```diff
@@ -54,6 +55,22 @@ def init_grid(self, xmin, xmax, zmin, zmax, nx, nz, max_grid_size=4):
         self._set_geom_str()
         self._set_grid_params()
 
+    def init_diags(self):
```

Comment:
When is this intended to be called? It's not, at present, called at any specific point.

I'd also like there to be some user flexibility in this specification, so that you could even set the step_interval to 10*max(diag_periods) or something if desired.